patternlab/styleguide/source/_data/data.json

```diff
@@ -5,6 +5,6 @@
   "bodyClass": "",
   "directory": "assets",
   "googleLanguages": "ar,es,fr,pt,ht,it,km,ko,ru,vi,zh-CN",
-  "sealImage": "../../assets/images/stateseal.png",
+  "sealImage": "/assets/images/stateseal.png",
```

Comment:
is this using the mayflower-assets package?

Reply:
oh it is nvm

Reply:
Yes